src/overlays/actors/ovl_Dm_An/z_dm_an.c

```diff
@@ -110,66 +121,85 @@ s32 DmAn_ChangeAnim(DmAn* this, PlayState* play, s32 animIndex) {
     return didAnimChange;
 }
 
-void func_80C1C5B4(DmAn* this) {
-    if (DECR(this->unk_2BA) == 0) {
-        this->unk_2B8++;
-        if (this->unk_2B8 >= 4) {
-            this->unk_2BA = Rand_S16Offset(30, 30);
-            this->unk_2B8 = 0;
+typedef enum DmAnEyes {
+    /* 0 */ DMAN_EYES_OPEN,
+    /* 1 */ DMAN_EYES_HALF1,
+    /* 2 */ DMAN_EYES_CLOSED,
+    /* 3 */ DMAN_EYES_HALF2,
+    /* 4 */ DMAN_EYES_COMFORT,
+    /* 5 */ DMAN_EYES_SAD,
+    /* 6 */ DMAN_EYES_RELIEVED_CLOSED,
```

Comment:
Was comfort previously named relieved? Kinda looks like the two are linked but not named the same anymore, or are they different?

Reply:
comfort and relieved are two different eyes.
Naming the eyes was difficult because they are used for expressing different things